src/global/functions.scss

```diff
@@ -0,0 +1,3 @@
+@function prem($size) {
```

Comment:
I was wondering, why you changed this from pxToRem to prem? Personally I find the first much clearer but you will certainly have given this some thought.

Reply:
Thought it was shorter to use prem instead of pxToRem, since we will write it a lot ;) but i can rename it to pxToRem to make it clearer